src/redis_cmd.cc

```diff
@@ -564,17 +560,17 @@ class CommandSetEX : public Commander {
 class CommandPSetEX : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
-    try {
-      auto ttl_ms = std::stol(args[2]);
-      if (ttl_ms <= 0) return Status(Status::RedisParseErr, errInvalidExpireTime);
-      if (ttl_ms > 0 && ttl_ms < 1000) {
-        ttl_ = 1;
-      } else {
-        ttl_ = ttl_ms / 1000;
-      }
-    } catch (std::exception &e) {
+    auto parseResult = ParseInt<long int>(args[2], 10);
```

Comment:
```suggestion
    auto parseResult = ParseInt<int>(args[2], 10);
```
`ttl_` is typed `int`

Reply:
`Long int` and `int` are not equal on some machines. Do you have any good suggestions about this?

Reply:
Maybe we can just use int64_t, because the next code will check the range.
`
if (ttl_ms > 0 && ttl_ms < 1000) {
      ttl_ = 1;
    } else {
      ttl_ = ttl_ms / 1000;
    }
`